src/sentry/conf/server.py

```diff
@@ -934,12 +934,12 @@ def create_partitioned_queues(name):
 }
 
 # Web Service
-SENTRY_WEB_HOST = "localhost"
+SENTRY_WEB_HOST = "127.0.0.1"
```

Comment:
Maybe this one can stay as browsers has a workaround for this? (unless the issue you mention strictly maps `localhost` to `::1` instead of both that and `127.0.0.1` which I'd find odd)

Reply:
Maybe. I also just don’t think it’s worth it. Connecting over localhost shouldn’t matter if it’s ipv4 or ipv6. So let’s just make sure it’s ip4. And granted, this variable is used for listening and binding. So it’s kinda odd to bind to a hostname anyways and not an IP.